signalfx/config/signalfx.config.pb.html

```diff
@@ -94,7 +94,10 @@ <h2 id="Params">Params</h2>
 <td><code>string</code></td>
 <td>
 <p>Required. The access token for the SignalFx organization that should
-receive the metrics.</p>
+receive the metrics.  This can also be configured via an environment
+variable SIGNALFX<em>ACCESS</em>TOKEN set on the adapter process which makes it
```

Comment:
Seems like the conversion from the proto to the HTML didn't parse correctly. Should you have included backquotes around it?

```
variable `SIGNALFX_ACCESS_TOKEN` set on ...
```